zabbix.yaml

```diff
@@ -1,5 +1,7 @@
 ---
-:zabbix_host: zabbix.domain.tld
+:zabbix_host:
+  - zabbix1.domain.tld
+  - zabbix2.domain.tld
 :zabbix_port: 10051
```

Comment:
Having multiple hosts is also means we need to be able to set `port` per each host, no?
You could wrap this with separate field, like

```yaml
:zabbix_hosts:
- address: zabbix1.domain.tld
  port: 10051
- address: zabbix2.domain.tld
  port: 10051
```

- The port can be optional and can inherit from `:zabbix_port` or default to `10051`
- To keep backward compatibility, you keep `:zabbix_host` and `:zabbix_port`, and in the report logic you just convert those to the newer structure